vent/controller/control_module.py

```diff
@@ -42,7 +42,8 @@ class ControlModuleBase:
     """
 
     def __init__(self):
-
+        self.logger = logging.getLogger(__name__)
```

Comment:
i think you might be going for hierarchal loggers inheriting handling here, but i think `getLogger()` with no args sets `name` to `'root'` so `__name__` here (`vent.controller.control_module`) wouldn't inherit.